packages/vertexai/src/models/generative-model.ts

```diff
@@ -82,7 +83,17 @@ export class GenerativeModel {
         project: vertexAI.app.options.projectId,
         location: vertexAI.location
       };
-      if ((vertexAI as VertexAIService).appCheck) {
+
+      if (
+        vertexAI.app &&
```

Comment:
Can there be a vertexAI instance that doesn't have an `app` property? How does that happen? Actually come to think of it, what do you think about putting `a` ? here https://github.com/firebase/firebase-js-sdk/blob/main/packages/app/src/internal.ts#L173 so that it can handle being passed undefined/null, so it would cover if any other caller happens to pass it that.

Reply:
To answer your first question, I'm not sure. I don't think it's possible but the rest of the code (above) checks for it.

As for the latter, I'll add the ability for the function to take null / undefined!

Reply:
Done.